### PR TITLE
update sponsors and donations

### PR DESCRIPTION
**Pull request by narimiran**
@dom96, please take a look if everything is ok.

### CODE REVIEW

File: jekyll/_data/sponsors.csv [hunk 1]
```diff
@@ -1,60 +1,50 @@
 logo,name,url,this_month,all_time,since,level
-/assets/bountysource/status.png,Status,https://status.im,19000,19000,"Aug 7, 2018",250
-/assets/bountysource/xored.svg,"Xored Software, Inc.",http://www.xored.com,250,6500,"Jun 20, 2016",250
-"https://cloudinary-a.akamaihd.net/bountysource/image/gravatar/d_noaoqqwxegvmulwus0un.png,c_pad,w_400,h_400,b_white/2969bb7ed867ad934f35b1016d268ae1",Varriount,,251,5267,"Nov 18, 2016",0
-,skunkiferous,,40,920,"Oct 2, 2016",75
-,endragor,,25,700,"Apr 7, 2016",25
-,FedericoCeratto,,25,700,"Apr 7, 2016",25
-,"Adrian Veith",,25,700,"Apr 20, 2016",25
-,jcosborn,,25,525,"Nov 21, 2016",25
+/assets/bountysource/status.png,Status,,19000,19000,"Aug 7, 2018",250
```
Comment (dom96) [inline jekyll/_data/sponsors.csv]: Status' website is missing.

File: jekyll/_data/sponsors.csv [hunk 2]
```diff
@@ -1,60 +1,50 @@
...
+/assets/bountysource/xored.svg,"Xored Software, Inc.",http://www.xored.com,250,9250,"Jun 20, 2016",250
```
Comment (dom96) [inline jekyll/_data/sponsors.csv]: Are we certain Xored is still contributing?

Comment (narimiran) [reply jekyll/_data/sponsors.csv]: Based on the total amount change between the last update (11 months ago) and now, it is exactly $250 per month still.

File: jekyll/_data/sponsors.csv (hunk 1 above)
Comment (narimiran) [reply jekyll/_data/sponsors.csv]: Will add it.

**Pull request merged**